beetsplug/vocadb.py

```diff
@@ -162,11 +160,27 @@ class AlbumFindResultDict(FindResultDict):
     items: list[AlbumDict]
 
 
+class ConfigDict(TypedDict):
+    prefer_romaji: bool
+    translated_lyrics: bool
+    include_featured_album_artists: bool
+    va_string: str
+    max_results: int
```

Comment:
```suggestion
    max_results: str
```
See my comment in `default_config`.